hack/run-preflight-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
instead of this script, update `hack/run-integration-tests.sh` and reuse. Preflight tests should be executed in particular ns in CI cluster, not `default` ns.